site-service/src/test/java/uk/ac/ox/ndph/mts/site_service/controller/SiteControllerTests.java

```diff
@@ -29,7 +29,7 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
-@SpringBootTest(properties = { "spring.cloud.config.discovery.enabled = false" , "spring.cloud.config.enabled=false", "server.error.include-message=always", "spring.main.allow-bean-definition-overriding=true" })
+@SpringBootTest(properties = { "spring.cloud.config.discovery.enabled = false" , "spring.cloud.config.enabled=false", "server.error.include-message=always", "spring.main.allow-bean-definition-overriding=true", "fhir.uri=http://localhost:8080" })
```

Comment:
looks like saying this active profile is "local" will also remove many of the settings here.

Reply:
decided to keep as is. fix in a separate user story